pandas/core/generic.py

```diff
@@ -3388,6 +3389,9 @@ def align(self, other, join='outer', axis=None, level=None, copy=True,
         limit : int, default None
         fill_axis : {0, 1}, default 0
             Filling axis, method and limit
+        broadcast_axis : {0, 1}, default None
```

Comment:
use ``%(axes)s`` here so the doc-string fills this in. hmm, though this should actually be a ``_shared_doc``